src/main/kotlin/eu/europa/ec/eudi/sdjwt/SdJwtVerifier.kt

```diff
@@ -74,6 +75,11 @@ sealed interface VerificationError {
      * @param disclosures The disclosures for which there are no digests
      */
     data class MissingDigests(val disclosures: List<Disclosure>) : VerificationError
+
+    @JvmInline
+    value class Cause(val value: String) : VerificationError {
```

Comment:
@babisRoutis Don't really like the name to be honest. I believe something like `Unexpected` would make more sense.

Reply:
Rename to `Other`

Reply:
Certainly, it is not used to wrap unexpected exceptions